### DISCUSSION TITLE
token.json on raspberry pi zero

### DISCUSSION THREAD

**Discussion by IvoHajducek** [skuethe/MMM-Spotify#135] [Q&A]
Hello, 
 I have raspberry pi zero with chromium in kiosk mode. I was able to login to Spotify account via browser and save cookies. Just FYI I've tried first auth script, but there was an error with missing captcha servise, which prevents login. Might be that on Zero it wan't work due performance, but I like to try. I'd like to ask how token.json should look like and are cookies sufficient to create json file?
.accounts.spotify.com	__Host-device_id
.accounts.spotify.com	__Secure-TPASESSION
.accounts.spotify.com	inapptestgroup
accounts.spotify.com	__bon
accounts.spotify.com	remember
.accounts.spotify.com	csrf_token
.accounts.spotify.com	sp_ac
.spotify.com	sp_dc
.spotify.com	sp_key

Thank you in advance

**Comment by skuethe** [comment]
Hi there. Sorry for the late reply, I wasn't tracking new discussions on this repo.
The token has the following format:
```json
{
    "access_token":"[REMOVED]",
    "token_type":"Bearer",
    "expires_in":3600,
    "refresh_token":"[REMOVED]",
    "scope":"playlist-read-private app-remote-control streaming user-modify-playback-state user-read-playback-state user-read-private",
    "expires_at":1631658194090
}
```
